pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, python3
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-details-directive";
+  version = "0.1.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "eL1qZ/eGohhoq/DmpZczQNfnpv1xsYkN6chW+Sh3s4s=";
+  };
+
+  propagatedBuildInputs = [
+    sphinx
+  ];
+
+  doCheck = false;
```

Comment:
Why is this disabled?

Reply:
The `sphinxcontrib-*` file I copied from as a template had it. Removed.